fvm/src/call_manager/mod.rs

```diff
@@ -121,6 +123,9 @@ pub trait CallManager: 'static {
         self.gas_tracker_mut().apply_charge(charge)?;
         Ok(())
     }
+
+    /// TODO
+    fn validate(&mut self, msg: Message, sig: Signature);
```

Comment:
Why do we have a signature here as a parameter? The whole point of account abstraction is that the signature is opaquely inside the message (if the validation logic is signature driven)

Reply:
`Message` represents and unsigned message, so the signature is deserialized elsewhere (in user code i believe since `Signature` is only used for the SDK validate_signature) and passed in. Would the signature be directly deserialized from `params` then? from what I can tell, the FVM does nothing automatically validate a signature, which would mean mostly just adding some internal functions to kick off a WASM module when it gets a signature type `External`.